app/models/admin_user.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: admin_users
+#
+#  id                     :integer          not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  sign_in_count          :integer          default(0), not null
+#  current_sign_in_at     :datetime
+#  last_sign_in_at        :datetime
+#  current_sign_in_ip     :inet
+#  last_sign_in_ip        :inet
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+class AdminUser < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, 
```

Comment:
Why is this repeated several times?

Reply:
I don't know, but since it's not even using the admin user for login or anything, it would be better to remove it altogether, right?

Reply:
You've created a `admin_users` table that uses it. If you don't need that table, then remove it and this as well.